src/Microsoft.DotNet.Darc/DarcLib/Helpers/ManifestHelper.cs

```diff
@@ -102,4 +113,73 @@ List<string> GetTargetPaths(DownloadedAsset asset)
         }
         return JObject.FromObject(manifestObject, JsonSerializer.CreateDefault(new JsonSerializerSettings() { NullValueHandling = NullValueHandling.Ignore }));
     }
+
+    private static List<DownloadedAsset> SelectMergedManifestAssets(IEnumerable<DownloadedBuild> downloadedBuilds)
+    {
+        return downloadedBuilds.Select(build =>
+            build.DownloadedAssets.Where(asset =>
+                asset.Asset.Name.EndsWith(MergedManifestFileName)))
+            .SelectMany(x => x).ToList();
```

Comment:
nit: For Linq, it's a custom that you format something like this:
```suggestion
        return downloadedBuilds
            .Select(build => build.DownloadedAssets
                .Where(asset => asset.Asset.Name.EndsWith(MergedManifestFileName)))
            .SelectMany(x => x)
            .ToList();
```

so that each transformation is on a new line.

Reply:
Thanks. Updated.